src/Development/IDE/LSP/LanguageServer.hs

```diff
@@ -107,6 +109,8 @@ runLanguageServer options userHandlers getIdeState = do
             , LSP.onStartup = handleInit (signalBarrier clientMsgBarrier ()) clearReqId waitForCancel clientMsgChan
             }
 
+    LSP.setupLogger (Just "/tmp/lsp-hello.log") [] L.DEBUG
```

Comment:
I assume this is the code that makes it draft?

Reply:
That’s part of it but the main reason is that the changes to `haskell-lsp` aren’t in a mergable state as I only implemented the part that we rely on.

Reply:
I hope you wouldn't use hardcoded path after merge. On Windows it may produce some strange behaviour, see [similar situation here](https://github.com/baverman/vial/issues/7)

Reply:
I definitely won’t, there is a reason this is a draft PR :slightly_smiling_face: